error: C compiler cannot create executables

### Environment: (where does this bug occur, have you tried other environments)
eicweb/eic_dev_cuda:26.01.0-stable

### Steps to reproduce: (give a step by step account of how to trigger the bug)
```
. /opt/spack/share/spack/setup-env.sh
spack env deactivate
spack env create --without-view myenv
spack -e myenv install --add libsigsegv
```

### Expected Result: (what do you expect when you execute the steps above)
no errors

### Actual Result: (what do you get when you execute the steps above)
```
> spack -e myenv install --add libsigsegv
==> Warning: The following issues were ignored while updating the indices of binary caches
  Multiple errors during fetching:
        Error 1: FetchIndexError: Could not fetch manifest from https://ghcr.io/v2/eic/spack-v2025.11.0/manifests/index.spack, due to: GET https://ghcr.io/v2/eic/spack-v2025.11.0/manifests/index.spack returned 404: Not Found
        Error 2: FetchIndexError: Could not fetch manifest from https://ghcr.io/v2/eic/spack-v2025.11.0/manifests/index.spack, due to: GET https://ghcr.io/v2/eic/spack-v2025.11.0/manifests/index.spack returned 404: Not Found
==> Fetching https://ghcr.io/v2/spack/bootstrap-buildcache-v1/blobs/sha256:af305366a56ff47603977201d86a296cfaf6f5e6ed38412ecd68efb3e8ad9799
==> Fetching https://ghcr.io/v2/spack/bootstrap-buildcache-v1/blobs/sha256:97f8ea17f3df3fb38904450114cbef9b4b0ea9c94da9de7a49b70b707012277a
==> Installing "clingo-bootstrap@=spack~docs+ipo+optimized+python+static_libstdcpp build_system=cmake build_type=Release generator=make patches:=bebb819,ec99431 arch=linux-centos7-x86_64" from a buildcache
spack ==> Concretized 1 spec
 -   5rhe2fi  libsigsegv@2.14 build_system=autotools arch=linux-ubuntu24.04-x86_64_v2 %c=gcc@13.3.0
[-]  toh4tbn      ^compiler-wrapper@1.0 build_system=generic arch=linux-ubuntu24.04-x86_64_v2 
[e]  3ragg6b      ^gcc@13.3.0~binutils+bootstrap~graphite~mold~nvptx~piclibs~profiled~strip build_system=autotools build_type=RelWithDebInfo languages:='c,c++,fortran' arch=linux-ubuntu24.04-x86_64_v2 
 -   okmod5x      ^gcc-runtime@13.3.0 build_system=generic arch=linux-ubuntu24.04-x86_64_v2 
[e]  h5bqx6h      ^glibc@2.39 build_system=autotools arch=linux-ubuntu24.04-x86_64_v2 
[^]  hukkuds      ^gmake@4.4.1~guile build_system=generic arch=linux-ubuntu24.04-x86_64_v2 %c=gcc@13.3.0
[e]  prgoex6          ^gcc@13.3.0~binutils+bootstrap~graphite~mold~nvptx~piclibs~profiled~strip build_system=autotools build_type=Release languages:='c,c++,fortran' arch=linux-ubuntu24.04-x86_64_v2 
[^]  k4zluw6          ^gcc-runtime@13.3.0 build_system=generic arch=linux-ubuntu24.04-x86_64_v2 

==> Fetching https://ghcr.io/v2/eic/spack-v2025.11.0/blobs/sha256:70537741e0a458e6ae50ca5299ff0707a5206f7d6e4e75ae57d3d92e2c9d533d
==> Fetching https://ghcr.io/v2/eic/spack-v2025.11.0/blobs/sha256:3f68adf85b6309d47400ccc9ddd4a9d64cc77a228ed363e22abe0f62fefbb3d5
==> Extracting compiler-wrapper-1.0-toh4tbny4yqcfc3njsuusafoayzw7cyr from binary cache
==> Warning: compiler-wrapper@1.0 build_system=generic arch=linux-ubuntu24.04-x86_64_v2/toh4tbn is being registered in the database with prefix /root/spack/linux-x86_64_v2/compiler-wrapper-1.0-toh4tbny4yqcfc3njsuusafoayzw7cyr, but this directory does not contain an installation of the spec, due to: Install prefix /root/spack/linux-x86_64_v2/compiler-wrapper-1.0-toh4tbny4yqcfc3njsuusafoayzw7cyr does not exist.
==> compiler-wrapper: Successfully installed compiler-wrapper-1.0-toh4tbny4yqcfc3njsuusafoayzw7cyr
  Search: 0.00s.  Fetch: 0.26s.  Install: 0.03s.  Extract: 0.01s.  Relocate: 0.02s.  Total: 0.29s
[+] /opt/software/linux-x86_64_v2/compiler-wrapper-1.0-toh4tbny4yqcfc3njsuusafoayzw7cyr
==> Installing compiler-wrapper-1.0-toh4tbny4yqcfc3njsuusafoayzw7cyr [1/7]
[+] /usr (external gcc-13.3.0-3ragg6b3umoyiyncrxzuu4av737vk5ny)
[+] /usr (external glibc-2.39-h5bqx6hxhj2vl47md2a632ybomueuvu4)
==> No binary for gcc-runtime-13.3.0-okmod5xyzp7wqzlnby6uki5ax3r5kqqi found: installing from source
==> Installing gcc-runtime-13.3.0-okmod5xyzp7wqzlnby6uki5ax3r5kqqi [4/7]
==> No patches needed for gcc-runtime
==> gcc-runtime: Executing phase: 'install'
==> gcc-runtime: Successfully installed gcc-runtime-13.3.0-okmod5xyzp7wqzlnby6uki5ax3r5kqqi
  Stage: 0.00s.  Install: 0.04s.  Post-install: 0.04s.  Total: 0.12s
[+] /root/spack/linux-x86_64_v2/gcc-runtime-13.3.0-okmod5xyzp7wqzlnby6uki5ax3r5kqqi
[+] /opt/software/linux-x86_64_v2/gcc-runtime-13.3.0-k4zluw6pggqmoy4qf6qlalyiewaq6pdn
[+] /opt/software/linux-x86_64_v2/gmake-4.4.1-hukkudsxuqj3uxvi3kt55eghowv2irzm
==> No binary for libsigsegv-2.14-5rhe2fiasd353ux6ihmhvnpqm2wmrgvf found: installing from source
==> Installing libsigsegv-2.14-5rhe2fiasd353ux6ihmhvnpqm2wmrgvf [7/7]
==> Fetching https://mirror.spack.io/_source-cache/archive/cd/cdac3941803364cf81a908499beb79c200ead60b6b5b40cad124fd1e06caa295.tar.gz
    [100%]  464.66 KB @    4.2 MB/s
==> No patches needed for libsigsegv
==> libsigsegv: Executing phase: 'autoreconf'
==> libsigsegv: Executing phase: 'configure'
==> Error: ProcessError: Command exited with status 77:
    '/tmp/root/spack-stage/spack-stage-libsigsegv-2.14-5rhe2fiasd353ux6ihmhvnpqm2wmrgvf/spack-src/configure' '--prefix=/root/spack/linux-x86_64_v2/libsigsegv-2.14-5rhe2fiasd353ux6ihmhvnpqm2wmrgvf' '--enable-shared'

2 errors found in build log:
     14    checking for a race-free mkdir -p... /usr/bin/mkdir -p
     15    checking for gawk... gawk
     16    checking whether make sets $(MAKE)... yes
     17    checking whether make supports nested variables... yes
     18    checking for gcc... /opt/software/linux-x86_64_v2/compiler-wrapper-1.0-toh4tbny4yqcfc3njsuusafoayzw7cyr/libexec/spack/gcc/gcc
     19    checking whether the C compiler works... no
  >> 20    configure: error: in `/tmp/root/spack-stage/spack-stage-libsigsegv-2.14-5rhe2fiasd353ux6ihmhvnpqm2wmrgvf/spack-src':
  >> 21    configure: error: C compiler cannot create executables
     22    See `config.log' for more details

See build log for details:
  /tmp/root/spack-stage/spack-stage-libsigsegv-2.14-5rhe2fiasd353ux6ihmhvnpqm2wmrgvf/spack-build-out.txt
```

--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -530,7 +530,7 @@ user_spack_environment:
     - spack add edm4eic@git.main cxxstd=20
     - spack concretize
     - spack install --only dependencies edm4eic
-    - ls -al /opt/software/linux-*/compiler-wrapper-1.0-*/libexec/spack/gcc/g++
+    - ls -al $HOME/spack/linux-*/compiler-wrapper-1.0-*/libexec/spack/gcc/g++
     - spack install --verbose
     - ls $HOME/spack/
 

--- a/spack.sh
+++ b/spack.sh
@@ -15,6 +15,7 @@ a462612b64e97fa7dfe461c32c58553fd6ec63c5
 292b0dcaba3b2a5e3f9668d205d39fee2c715721
 d55f9abacc4e153d43a4c8bff81a383ba2f311f5
 678e506a95b319c573ba7e84703b06d7275ab80e
+52bef0e63711a0b63816b3f4b1561df5b158db65
 ---
 ## Optional hash table with comma-separated file list
 read -r -d '' SPACK_CHERRYPICKS_FILES <<- \
@@ -27,3 +28,4 @@ read -r -d '' SPACK_CHERRYPICKS_FILES <<- \
 ## 292b0dcaba3b2a5e3f9668d205d39fee2c715721: fix: write created time field with OCI buildcache config
 ## d55f9abacc4e153d43a4c8bff81a383ba2f311f5: views: collapse unique subtrees in symlink case
 ## 678e506a95b319c573ba7e84703b06d7275ab80e: fix: don't map prefix to view root for pkgs excluded from view
+## 52bef0e63711a0b63816b3f4b1561df5b158db65: fix: install missing upstream packages in local store

